src/app/shared/components/template/services/template-asset.service.ts

```diff
@@ -74,6 +78,25 @@ export class TemplateAssetService extends AsyncServiceBase {
     return value;
   }
 
+  private getFilePath(assetName, contentsEntry: IContentsEntryMinimal | Partial<IAssetEntry>) {
+    if (environment.deploymentConfig.supabase.enabled) {
```

Comment:
As well as `environment.deploymentConfig.supabase.enabled`, this should also be checking `appConfig.ASSET_PACKS.enabled`.
The separation makes sense as we move forward I think (we may want to expand supabase integration soon to cover functionality other than remote assets, e.g. auth).
So do we check both (requiring a subscription to `appConfigService.appConfig$`)? We're unlikely to want to change remoteAssetsEnabled at runtime, so it could just be another setting in environment.deploymentConfig?

Reply:
nit(blocking)
Agreed asset pack enabling doesn't make much sense to toggle during runtime so let's assume no subscriptions required. 

I think better than relying on checking supabase setting here we should probably expose a method to update/alter the `assetsContentList` and do so following download (linked to remote-asset system), so that this service doesn't need any awareness of supabase or cached file paths